kolla/node_custom_config/grafana.ini

```diff
@@ -0,0 +1,21 @@
+[server]
+root_url = "https://{{kolla_external_fqdn}}:3000"
+
+[auth]
+disable_login_form = "true"
+oauth_auto_login = "true"
+
+[auth.generic_oauth]
+name = Chameleon
+icon = signin
+enabled = true
+allow_sign_up = true
+client_id = "{{ keystone_idp_client_id }}"
+scopes = openid profile email
+auth_url = "https://auth.chameleoncloud.org/auth/realms/chameleon/protocol/openid-connect/auth"
+token_url = "https://auth.chameleoncloud.org/auth/realms/chameleon/protocol/openid-connect/token"
+api_url = "https://auth.chameleoncloud.org/auth/realms/chameleon/protocol/openid-connect/userinfo"
+
+role_attribute_path = contains(projects[*].id, 'Chameleon') && 'Admin' || contains(projects[*].id, 'CHI-210828') && 'Editor'
```

Comment:
what is the significance of 210828 here? Is this a project for associate site operators?

Reply:
Actually, can this be templated? It seems strange to have this hardcoded.

Reply:
exactly, see the note below. 210828 is from indyscc. I think the way to go would be to have a project for associate site operators, or perhaps one per site, similar to `Chameleon`. Perhaps "ops-<site_name>"